tests/packagedcode/data/plugin/help.txt

```diff
@@ -552,13 +552,6 @@ Package type:  mozilla
   description:       Mozilla XPI extension
   path_patterns:    '*.xpi'
 --------------------------------------------
-Package type:  msi
```

Comment:
Revert these changes, these plugins were not installed in your system (because it's a mac and improperly installed?) does not mean these are not there.
Do not do a `pip install` for your development, go a source checkout installation: https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#installation-from-source-code-git-clone

Reply:
Resolved!

Reply:
Nope, not resolved. Still present.

Reply:
Check for the (outdated) label on these conversations, when these are not there it means you've not changed anything in the line where the comment was there. Or you can just see the diff view.